### ISSUE TITLE
Questions

### ISSUE THREAD

**Issue by hikeeba** [rehanvdm/cdk-express-pipeline#6]
Just tested out this package and it looks very promising. Couple of questions if you have a moment.

* Do you expect this project/package to be expanded and supported for a while?
* What is the recommended pattern for handling multiple development/deployment environments (dev, test, staging, prod, etc.)?
  * We have been using https://github.com/cdklabs/cdk-pipelines-github, and it has stages that we use for this purpose.
* Assuming there is a pattern for multiple environments, is there also a pattern for passing configuration values per environment?
  * For example, a dev database may have a single instance while a prod one may have multiple for FT/HA.

Thanks so much!

**Comment by hikeeba** [comment]
Also, are you open to pull requests from other parties?

**Comment by rehanvdm** [comment]
Hey, appreciate the questions:

> Do you expect this project/package to be expanded and supported for a while?

100% This is not a short-term project and I hope that it becomes a strong contender to AWS CDK Pipelines

> What is the recommended pattern for handling multiple development/deployment environments (dev, test, staging, prod, etc.)? 

This is how I approach it, have a look at the "Configuration" and  "Multiple environments" sections on this post I wrote: https://rehanvdm.com/blog/aws-cdk-starter-configuration-multiple-environments-cicd (here is the attached GH repo for that: https://github.com/rehanvdm/cdk-starter). 

Basically, I like to create an `EnvironmentConfig` then and make a file for each environment config, like `config/development.ts`. Then in the CDK app select the correct environment config file depending on a Context Variable  (`-c`). 

> Assuming there is a pattern for multiple environments, is there also a pattern for passing configuration values per environment? 

Similar to the above, it leverages the CDK functionality, this package of mine is just a wrapper to improve the order of deployment for CDK stacks. So yes, if you can do it CDK then you can do it here as well.

**Comment by rehanvdm** [comment]
> Also, are you open to pull requests from other parties?

😃 YES, please! I see I don't have a CONTRIBUTING.md file. Just open an Issue first, then we can see if there are no one else working on it and I could give some tips. We then discuss ans assign it to you and you can start.

**Comment by hikeeba** [comment]
Perfect, thanks! We can close this issue and I'll try out the patterns mentioned in your post.
